src/parser/parseTransformAttribute.ts

```diff
@@ -87,8 +90,5 @@ export function parseTransformAttribute(attributeValue: string): TMat2D {
     matrices.push(matrix);
   }
 
-  return matrices.reduce(
-    (acc, matrix) => multiplyTransformMatrices(acc, matrix),
-    iMatrix
-  );
```

Comment:
we are changing a reduce to reduce right here nad i don't understand why.
I m incline to say don't touche the current transformation code at all, is not a cleanup anymore at that point

Reply:
I agree.
Touched it and became uneasy but the math is correct and I double checked that there is a test to cover it.
This proves it is all right: https://en.wikipedia.org/wiki/Matrix_multiplication#Associativity

Notice the args order passed to `multiplyTransformMatrices`